apps/emqx_management/src/emqx_mgmt_data_backup.erl

```diff
@@ -666,7 +666,10 @@ look_up_file(Filename) ->
         end,
     case lists:filter(Filter, backup_files()) of
         [] ->
-            {error, not_found};
```

Comment:
`delete_backup_file` also uses the `look_up_file`, maybe this change is less safe?

Reply:
I've changed it to accept other files only in the `check_import_json` here: https://github.com/emqx/emqx/pull/7996/commits/290fa929b2927c777a9894cdc2b96a0cd7b1a6ea